.github/workflows/qa.yml

```diff
@@ -0,0 +1,31 @@
+name: QA
+
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+
+jobs:
+  lint:
+    name: Rubocop
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - uses: ruby/setup-ruby@v1
+        with:
+          ruby-version: '2.7'
```

Comment:
FYI support for 2.7 [ends in March](https://endoflife.date/ruby), suggest testing with 3.1 unless the code doesn't work with it.